pkg/client/README.md

```diff
@@ -0,0 +1,132 @@
+# William
+
+This library aims to prevent writing to much code using william on your application.
+
+## Creating a service
+```go
+wi := william.New("URL OF WILLIAM", "ServiceName")
+```
+You can set the HTTP Client that will made the request also the tracing if anyone. (if not set nil)
```

Comment:
Suggestion: "(...) will made the request. You can also configure tracing (optional, set nil to disable it)."

Reply:
This example is out of date. Thanks.